cql3/type_json.cc

```diff
@@ -489,7 +489,7 @@ struct to_json_string_visitor {
     sstring operator()(const string_type_impl& t) { return quote_json_string(t.to_string(bv)); }
     sstring operator()(const bytes_type_impl& t) { return quote_json_string("0x" + t.to_string(bv)); }
     sstring operator()(const boolean_type_impl& t) { return t.to_string(bv); }
-    sstring operator()(const timestamp_date_base_class& t) { return quote_json_string(t.to_string(bv)); }
+    sstring operator()(const timestamp_date_base_class& t) { return quote_json_string(timestamp_to_json_string(t, bv)); }
```

Comment:
But neither of these changes are critical. Anyway, now I'm waiting for CI to pass (I started it now).

Reply:
> Looks good, just one thought: If the function is called timestamp_to_json_string, maybe it should return the full JSON string, i.e., with the double quotes? It can either call quote_json_string itself like you did, or just add the two quotes in the beginning and the end (there is no need to escape anything because you know you didn't use any special characters).

I thought about that but decided to keep code's common style: when we wrapping strings in quotes in visitor class.
I could do it just inside function if you think it's better.

> Another nitpick: Since this new function timestamp_to_json_string() is used only in this source file, maybe its definition can be in this file and not a different file (and it can be a static function)? Or it needs non-public APIs to be implemented?

Oh, that's would be much better, I agree. And I wanted to do that way. But this function uses things from different module so we should export something anyway. Better of two evils...
I could try implement dependent functionality just in place if you think it would be better